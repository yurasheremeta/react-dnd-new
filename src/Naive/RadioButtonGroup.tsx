import * as React from 'react';

interface RadioButtonGroupProps {
    onChange: any
}
class RadioButtonGroup extends React.Component<RadioButtonGroupProps>{

    render(){
        
        return( 
            <form>
				<input value="1"  name= "radio" onChange={this.props.onChange} type="radio"/>
				<input value="2" name="radio" onChange={this.props.onChange} type="radio" />
				<input value="3" name="radio" onChange={this.props.onChange} type="radio" />
			</form>
            // <input type="radio" value={this.props.value} onChange={this.props.onChange}/>
        )
    }
}

export default RadioButtonGroup;