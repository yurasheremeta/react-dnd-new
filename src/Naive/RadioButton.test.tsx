import * as React from 'react';
import { shallow, mount} from 'enzyme';
import renderer from 'react-test-renderer';
import RadioButtonGroup from './RadioButtonGroup';
import 'jsdom-global/register';

describe('RadioButton tests' , () => {
    it('Render correct RadioButton group' , () => {
        const wrapper = renderer.create(<RadioButtonGroup  onChange={() => {}}/>).toJSON();
        expect(wrapper).toMatchSnapshot();
    })
})