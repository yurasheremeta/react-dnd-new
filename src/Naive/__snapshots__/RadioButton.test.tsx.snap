// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButton tests Render correct RadioButton group 1`] = `
<form>
  <input
    name="radio"
    onChange={[Function]}
    type="radio"
    value="1"
  />
  <input
    name="radio"
    onChange={[Function]}
    type="radio"
    value="2"
  />
  <input
    name="radio"
    onChange={[Function]}
    type="radio"
    value="3"
  />
</form>
`;
